### PR TITLE
fix: allow for saving insurance when country is empty

### LINKED ISSUE #7331: bug: saving existing insurance with no country in address
### Describe the bug

should be able to save an insurance without choosing the country of the subscriber
country is not required for billing in the u.s.

### To Reproduce

1. Add a '....'

### Client configuration

- Browser:
- OpenEMR version:
- Operating system:

## Upvote & Fund

- We're using [Polar.sh](https://polar.sh/openemr) so you can upvote and help fund this issue.
- We receive the funding once the issue is completed & confirmed by you.
- Thank you in advance for helping prioritize & fund our backlog.

<a href="https://polar.sh/openemr/openemr/issues/7331">
<picture>
  <source media="(prefers-color-scheme: dark)" srcset="https://polar.sh/api/github/openemr/openemr/issues/7331/pledge.svg?darkmode=1">
  Fund with Polar
</picture>
</a>

### PR DESCRIPTION
**Pull request by stephenwaite**
Fixes #7331

### CODE REVIEW

File: src/Billing/InsurancePolicyTypes.php [hunk 1]
```diff
@@ -18,7 +18,7 @@ class InsurancePolicyTypes
 {
     // Hard-coding this array because its values and meanings are fixed by the 837p
 // standard and we don't want people messing with them.
-    const POLICY_TYPE_NOT_APPLICABLE = '';
+    const POLICY_TYPE_NOT_APPLICABLE = 'FALSE';
```
Comment (adunsulag) [inline src/Billing/InsurancePolicyTypes.php:21]: So do we want to actually save a FALSE value in the database for this option?  Will this break legacy database entries?  You'd know this better than I do, just a bit confused on how we get to the FALSE option.

Comment (stephenwaite) [reply src/Billing/InsurancePolicyTypes.php:21]: was working on some strange data, after looking at demo data it appears the policy type is saved as an empty string, then thought maybe on upgrade we should change empty string to null and allow null to be `Yes`
Screenshot from 2024-04-09 17-45-21
Screenshot from 2024-04-09 17-47-33

Comment (stephenwaite) [reply src/Billing/InsurancePolicyTypes.php:21]: then thought this is pretty trivial :smile:

**Review by bradymiller — APPROVED** [review]

**Comment by bradymiller** [comment]
2DV

**Pull request merged**